src/js/claims-status/containers/AppealsV2StatusPage.jsx

```diff
@@ -24,6 +25,9 @@ class AppealsV2StatusPage extends React.Component {
   }
 
   render() {
+    if (this.props.appealsLoading) {
```

Comment:
I guess we'll need the same thing in the `Detail` page, in case someone navigates there directly. An alternative might be to (once my tabs PR gets merged in) do this in AppealInfo and conditionally render either a `LoadingIndicator` or `this.props.children`. That would also mean moving some of the appeal fetching logic up there as well. But we'll cross that bridge when we get there :)

Reply:
Yeah, and it probably makes the most sense to pull that fetching logic into `AppealInfo` anyhow. 👍